internal/component/pyroscope/write/write.go

```diff
@@ -590,3 +683,24 @@ func validateLabels(lbls labels.Labels) error {
 
 	return nil
 }
+
+func configureTracing(config Arguments, httpClient *http.Client) error {
+	if opentracing.IsGlobalTracerRegistered() {
+		return fmt.Errorf("opentracing was configured, but is not expected to be")
```

Comment:
```suggestion
		return fmt.Errorf("opentracing was configured, but is not expected to be")
```

Not super clear to me if this is a path users might face, but the action to be taken when seeing should be clearer. Maybe if it is impossible to configure alloy with opentracing we should just panic?

Maybe we can explain how this is related to those settings: https://grafana.com/docs/alloy/latest/reference/config-blocks/tracing/